rules/framework.bzl

```diff
@@ -294,8 +295,6 @@ def _apple_framework_packaging_impl(ctx):
         "multi_arch_linked_binaries",
         "multi_arch_dynamic_libraries",
         "source",
-        "define",
-        "include",
```

Comment:
QQ: looks like does `framework_includes` takes care of `include` - what about `define`?

Reply:
AFAIK `define` and `include` were migrated to the cc compilation context in 4.0.0:

* `3.7.0` docs: https://docs.bazel.build/versions/3.7.0/skylark/lib/ObjcProvider.html
* `4.0.0` docs: https://docs.bazel.build/versions/4.0.0/skylark/lib/ObjcProvider.html (`define` and `include` removed)

`--incompatible_objc_provider_remove_compile_info` is enabled by default in 4.0.0+. I don't know if we need to have something special in here for people who are still on earlier versions and don't have it set.